.github/workflows/build-test.yml

```diff
@@ -217,7 +217,7 @@ jobs:
       - name: Test Maven Java
         run: |-
           cd plugin-tester-java
-          mvn -Dakka.grpc.project.version=`cat ~/.version` akka-grpc:generate compile
```

Comment:
`akka-grpc:generate` should not be needed explicitly as it is already run bin `compile` (otherwise it would fail I assume).